lib/util/include/hse_util/bin_heap.h

```diff
@@ -155,4 +150,8 @@ bin_heap2_replace_src(struct bin_heap2 *bh, struct element_source *es);
 s64
 bin_heap2_age_cmp(struct element_source *es1, struct element_source *es2);
```

Comment:
What is the significance of the 2?

Reply:
bin_heap2 is a replacement for bin_heap.  For a while both bin_heap and bin_heap2 were in use, but we may have eliminated all uses of the original, in which case we can rename this to bin_heap.

Reply:
Right now bin_heap is used by compaction loops.